src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,195 @@
+
+- An `UPDATE` operation replicated to the destination cluster.
+- A cross-cluster write occurs, which means both clusters are writing to the same key. For example, if the LDR stream attempts to apply a row to the destination cluster where the existing row on the destination was not written by the LDR stream.
+
+LDR uses _last write wins (LWW)_ conflict resolution based on the [MVCC timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#mvcc) of the replicating write. LDR will resolve conflicts by inserting the row with the latest MVCC timestamp. When a conflict cannot apply, due to violating a foreign key constraint or schema constraint, it will be retried for up to a minute and then put in the [DLQ](#dead-letter-queue-dlq) if it could not be resolved. 
+
+### Dead letter queue (DLQ)
+
+When the LDR job starts, it will create a DLQ table with each replicating table so that unresolved conflicts can be tracked. The DLQ will contain the writes that LDR cannot apply after the retry period, which could occur if:
+
+- The destination table was dropped.
+- The destination cluster is unavailable.
+- Tables schemas do not match.
+
+In `validated` mode, rows are also sent to the DLQ when:
+
+- [Foreign key]({% link {{ page.version.version }}/foreign-key.md %}) dependencies are not met where there are foreign key constraints in the schema.
+- Unique indexes and other constraints are not met.
+
+{{site.data.alerts.callout_info}}
+LDR will not pause when the writes are sent to the DLQ, you must manage the DLQ manually.
+{{site.data.alerts.end}}
+
+To manage the DLQ, you can evaluate entries in the `incoming_row` column and apply the row manually to another table with SQL statements.
+
+As an example, for an LDR stream created on the `movr.public.promo_codes` table:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW TABLES;
+~~~
+
+The table will have a random number within its name to ensure it is unique.
+
+~~~
+    schema_name    |         table_name         | type  | owner | estimated_row_count | locality
+-------------------+----------------------------+-------+-------+---------------------+-----------
+  crdb_replication | dlq_113_public_promo_codes | table | node  |                 186 | NULL
+  public           | promo_codes                | table | root  |                1047 | NULL
+  public           | rides                      | table | root  |                 976 | NULL
+  public           | user_promo_codes           | table | root  |                 134 | NULL
+  public           | users                      | table | root  |                 424 | NULL
+  public           | vehicle_location_histories | table | root  |               13012 | NULL
+  public           | vehicles                   | table | root  |                 153 | NULL
+(7 rows)
+~~~
+
+The schema for the `movr.crdb_replication.dlq_113_public_promo_codes` DLQ:
+
+~~~sql
+CREATE TABLE crdb_replication.dlq_113_public_promo_codes (
```

Comment:
Shall we add a table to describe each of these columns?

Reply:
Yeah that would be awesome. Ill ask michael or someone from eng to.

Reply:
If you want this in before merging, can we get that today, otherwise let's create an issue for this and the DLQ query.